### PR TITLE
New package: apt-1.7.2

### PR DESCRIPTION
**Pull request by marmeladema**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3400,3 +3400,6 @@ libsnoresettings-qt5.so.0.7 snorenotify-0.7.0_1
 libTECkit.so.0 libteckit-2.5.8_1
 libTECkit_Compiler.so.0 libteckit-2.5.8_1
 libwf-config.so wf-config-0.1_1
+libapt-private.so.0.0 apt-1.2.29_1
```
Comment (maxice8) [inline common/shlibs]: please don't add unless using

File: srcpkgs/apt/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'apt'
+pkgname=apt
+version=1.7.2
+revision=1
+build_style=cmake
+configure_args="-DDPKG_DATADIR=/usr/share/dpkg -DCURRENT_VENDOR=debian -DDOCBOOK_XSL=/usr/share/xsl/docbook"
+hostmakedepends="pkg-config docbook-xsl doxygen po4a w3m libxslt"
+makedepends="db-devel gnutls-devel bzip2-devel liblzma-devel liblz4-devel libzstd-devel libseccomp-devel eudev-libudev-devel dpkg-devel"
```
Comment (maxice8) [inline srcpkgs/apt/template]: cut at 80 col

File: srcpkgs/apt/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'apt'
+pkgname=apt
+version=1.7.2
+revision=1
+build_style=cmake
+configure_args="-DDPKG_DATADIR=/usr/share/dpkg -DCURRENT_VENDOR=debian -DDOCBOOK_XSL=/usr/share/xsl/docbook"
```
Comment (maxice8) [inline srcpkgs/apt/template]: cut at 80 col

File: common/shlibs (hunk 1 above)
Comment (marmeladema) [reply common/shlibs]: All binaries provided by this package link against this library. Also if not specified xbps-src complains when building the package:
> => WARNING: apt-1.7.2_1: libapt-private.so.0.0 not found in common/shlibs!                                                                                                                                                                                     
> => WARNING: apt-1.7.2_1: libapt-inst.so.2.0 not found in common/shlibs!                                                                                                                                                                                        
> => WARNING: apt-1.7.2_1: libapt-pkg.so.5.0 not found in common/shlibs!

File: srcpkgs/apt/template (hunk 2 above)
Comment (marmeladema) [reply srcpkgs/apt/template]: Done

File: srcpkgs/apt/template (hunk 3 above)
Comment (marmeladema) [reply srcpkgs/apt/template]: Done

File: common/shlibs (hunk 1 above)
Comment (maxice8) [reply common/shlibs]: please don't add unless using

**Pull request closed without merging**